dev/com.ibm.ws.jca/src/com/ibm/ws/jca/internal/ConnectorModuleMetaDataImpl.java

```diff
@@ -46,6 +46,16 @@
 
     static final String RA_MODULE_CONSTANT = "ResourceAdapterModule";
 
+    /**
+     * Id for WebSphere JMS resource adapter
+     */
+    private static final String WASJMS = "wasJms";
```

Comment:
to make this variable more clear, we could do make it `Set<String> reservedIDs`, and then we can check `if(reservedIDs.contains(id))`, and if it does, pass in `reservedIDs` as arg `{1}`